src/opm/io/eclipse/ESmry.cpp

```diff
@@ -419,6 +419,12 @@ void ESmry::updatePathAndRootName(Opm::filesystem::path& dir, Opm::filesystem::p
     rootN = rootN.stem();
 }
 
+bool ESmry::write_rsm(const std::optional<std::string>& o_filename) const {
```

Comment:
Just call it `filename` -the leading `o_` was really funny.

Reply:
The exact name here will be adjusted as the method itself is completed; right now it's named `o_filename` so as not to be shadowed by the de-optionalised value `filename`, but that'll probably be turned into a `Opm::filename::path` or something as the method is completed, with a different name. Details TBD, basically.